test/e2e/scheduling/preemption.go

```diff
@@ -162,8 +162,8 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			PriorityClassName: highPriorityClassName,
 			Resources: &v1.ResourceRequirements{
 				Requests: podRes,
+				Limits:   podRes,
```

Comment:
Do we need `Limits`?

Reply:
Yeah, without them I got an error that non-over-committable resources need to have a limit set